specification/specs/x_pack/async_search/submit/AsyncSearchSubmitRequest.ts

```diff
@@ -25,6 +25,10 @@
 interface AsyncSearchSubmitRequest extends RequestBase {
   path_parts?: {
     index?: Indices
+    batched_reduce_size?: long
+    wait_for_completion_timeout?: Time
+    keep_on_completion?: boolean
+    typed_keys?: boolean
```

Comment:
Are you sure about this? I think this should go in `query_parameters`.

Reply:
Now that you mention it! Yes, they should be in `query_parameters`! The tests did pass though 😉

Reply:
feedback addressed

Reply:
> The tests did pass though 

Yes, unfortunately, we can't check if a parameter should go in the URL or in the query :S